module/x/gravity/types/ethereum_signer.go

```diff
@@ -21,6 +23,21 @@ func NewEthereumSignature(hash []byte, privateKey *ecdsa.PrivateKey) ([]byte, er
 	return crypto.Sign(protectedHash.Bytes(), privateKey)
 }
 
+// decodeSignature was duplicated from go-ethereum with slight modifications
+func decodeSignature(sig []byte) (r, s *big.Int, v byte) {
```

Comment:
We should drop the module changes -- we can't make potentially state-breaking changes in a minor release, the relayer check will suffice until an upgrade.